### DISCUSSION TITLE
Running in subdirectory on GitHub Actions

### DISCUSSION THREAD

**Discussion by lubo** [super-linter/super-linter#5732] [Q&A]
I'm trying to do so to work around https://github.com/python/mypy/issues/16070. I tried setting `DEFAULT_WORKSPACE` and `GITHUB_WORKSPACE` to `/github/workspace/<subdirectory>`, I also tried setting `RUN_LOCAL=true`, but nothing worked. Any idea on how to do this?

**Comment by ferrarimarco** [comment]
I think the [`working-directory` configuration directive is what you want](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory).

`DEFAULT_WORKSPACE` is to set the working directory inside the Super-linter container.

**Reply by lubo** [reply]
`working-directory` is incompatible with `uses`. I'm getting the following error:

>The workflow is not valid. .github/workflows/ci.yml (Line: 36, Col: 9): Unexpected value 'working-directory'

See https://github.com/lubo/useful-scripts/blob/f4e9beb143e880cabb4b23c212a20b6e81556127/.github/workflows/ci.yml and https://github.com/lubo/useful-scripts/actions/runs/9387339639.

That's why I'm interested in `DEFAULT_WORKSPACE` since `uses: super-linter/super-linter/slim@v6.5.1` runs a Docker container.

May you please reopen this discussion? I haven't received a satisfying answer.
